.github/workflows/ci.yml

```diff
@@ -83,3 +75,28 @@ jobs:
 
     # Fail if any changes were written to source files (ex, from: build/build-cdt-lib.js).
     - run: git diff --exit-code
+
+  smoke:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        invert: [false, true]
+      # e.g. if smoke 0 failes, continue with smoke 1 anyway
+      fail-fast: false
+    env:
+      # The smokehouse tests run by job smoke 0. smoke-1 will run the rest.
```

Comment:
whatcha thinking  with ToTChrome in the mix too?

Reply:
> whatcha thinking with ToTChrome in the mix too?

```yml
matrix:
  smokeTestInvert: [false, true]
  chrome: [stable, tot]
```
and then conditionally set `CHROME_PATH`? conditionals are awkward but that doesn't seem too bad

Reply:
I broke up the ToT and parallel jobs for a reason. Things are getting complex :)